src/netspresso_trainer/schedulers/builder.py

```diff
@@ -5,18 +5,14 @@
 
 
 def build_scheduler(optimizer, conf_training):
-    scheduler_name = conf_training.sched
+    conf_scheduler = conf_training.scheduler
+    scheduler_name = conf_scheduler.name
     num_epochs = conf_training.epochs
-    conf_sched = OmegaConf.create({
-        'min_lr': conf_training.min_lr,
-        'power': conf_training.sched_power,
-        'warmup_bias_lr': conf_training.warmup_bias_lr,
-        'warmup_iters': conf_training.warmup_epochs,
-        'total_iters': num_epochs,
-        'iters_per_phase': conf_training.iters_per_phase,  # TODO: config for StepLR
-    })
+
+    # Copy training num_epochs to sub-config scheduler
+    conf_scheduler.total_iters = num_epochs # fix total_iters as num_epochs
 
     assert scheduler_name in SCHEDULER_DICT, f"{scheduler_name} not in scheduler dict!"
-    lr_scheduler = SCHEDULER_DICT[scheduler_name](optimizer, **conf_sched)
+    lr_scheduler = SCHEDULER_DICT[scheduler_name](optimizer, conf_scheduler)
```

Comment:
I suggest changing `conf_scheduler` to `scheduler_conf` in my opinion, too.

```suggestion
    lr_scheduler = SCHEDULER_DICT[scheduler_name](optimizer, scheduler_conf)
```